### ISSUE TITLE
Add a proper splash and add a good font like Jetbrains Mono or Montserrat.

### ISSUE THREAD

**Issue by Adithya-Kalany** [aleksey-hoffman/sigma-file-manager#166]
## Feature request:
Instead of the current one, add a proper splash screen and add a good font like Jetbrains Mono or Montserrat.
Makes it look "professional"

**Comment by aleksey-hoffman** [comment]
I will add support for custom fonts in one of the future updates.

Though I'm not sure what you mean by "proper splash"? What makes the current one "not proper"?

**Comment by Adithya-Kalany** [comment]
![image](https://user-images.githubusercontent.com/69103102/139697996-1261e04e-4def-427e-9a88-ccc8a8e2e78b.png)
I just don't like default font and then yes, I would love it make it like a Adobe Photoshop kind of Splash Screen.

**Comment by Adithya-Kalany** [comment]
And you replied so quick :o

**Comment by Adithya-Kalany** [comment]
And when I try to open the app with these settings, it doesn't and I have to use CTRL+SHIFT+SPACE. I'm on Windows 11, the latest cumulative update.

**Comment by aleksey-hoffman** [comment]
> And when I try to open the app with these settings

Are you sure it's not caused by this setting?

![image](https://user-images.githubusercontent.com/61761672/139703095-055f1d51-1e2c-4399-9ff3-45e139bac4e5.png)

**Comment by Adithya-Kalany** [comment]
Yep; checked it properly and it is disabled.

**Comment by Adithya-Kalany** [comment]
Sorry for the very late response.

**Comment by aleksey-hoffman** [comment]
Thanks, I will try to fix it for the next update

**Comment by Adithya-Kalany** [comment]
Alrighty, it's not my side of issues right?

**Comment by aleksey-hoffman** [comment]
Yeah. I just tested it. It only re-opens the window from the tray menu

**Comment by Adithya-Kalany** [comment]
Alright, and one last question: how do I edit the code of this project?

**Comment by Adithya-Kalany** [comment]
Thanks a lot :D

**Comment by aleksey-hoffman** [comment]
Wait. I just remembered, I already fixed this problem in v1.5 version. It will work after the next update. 

If you want to learn how to code:

1. Install [VScode](https://code.visualstudio.com/) (code editor)
2. Download the project folder from the [main page](https://github.com/aleksey-hoffman/sigma-file-manager):

![image](https://user-images.githubusercontent.com/61761672/139713079-d968d312-fefa-45dd-8478-b8e1f2613b4b.png)
3. Download and install Node.js (version 15.14.0): https://nodejs.org/en/download/releases/
4. Watch a few Youtube tutorials about Javascript, Vue.js, and Electron

To open the app in "development" mode. Open the project folder in command prompt and then run command: `npm run electron:serve`

**Comment by Adithya-Kalany** [comment]
Alright, thanks a lot. It would help if you could guide me to the opening page of the app, as I want to experiment (i know Javascript and all) just wanna try. Thanks.

**Comment by aleksey-hoffman** [comment]
If you do the steps I mentioned above, the app will open after you run: `npm run electron:serve`
And then you will be able to modify the code and see the changes

**Comment by Adithya-Kalany** [comment]
Thanks a lot, really appreciate it 😄

**Comment by Adithya-Kalany** [comment]
![image](https://user-images.githubusercontent.com/69103102/139720298-015b39f6-6238-438d-8e03-0a707649a54a.png)
I did it, changed the title to Montserrat cuz it's my favorite. Never thought I could do :D

Thanks a lot
[index.txt](https://github.com/aleksey-hoffman/sigma-file-manager/files/7455317/index.txt)
.

**Comment by aleksey-hoffman** [comment]
Good job!
The main branch currently has some problems. So you can't really build and use the app, until I fix the problems and update the code on Github

**Comment by Adithya-Kalany** [comment]
I understood; just did it for fun. Thanks a lot for guiding me.
